portfolio/src/main/webapp/script.js

```diff
@@ -15,17 +15,34 @@
 /**
  * Adds a random fun fact about myself to the page.
  */
-function addRandomFact() {
+function addFunFact() {
   const facts =
       ['I have two older brothers!',
        'I\'ve played tennis for fourteen years!', 
        'My favorite book is Enlightenment Now by Steven Pinker.', 
        'My favorite TV show is It\'s Always Sunny in Philadelphia.'];
 
-  // Pick a random fact.
+  // Pick a random fun fact.
   const fact = facts[Math.floor(Math.random() * facts.length)];
```

Comment:
optional of course, but if you notice, when you click the "Learn something about me!" button multiple times, you'll often get the same fact in a row. Though this is working as intended, as the selection is random from a set list, it does give off the impression that the button isn't responding (that's actually what my first thought was). Might be cool to add a bit more logic to the selection!

Reply:
Ok, I added a change that prevents the random fun fact button from giving the same fact multiple times in a row. The implementation I used made one of the fun facts more likely to appear than another (because if you get the same fun fact twice, it will move deterministically to the fun fact "in front of" the previous one), but it is difficult to tell this from the user perspective.